pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -17879,8 +17879,8 @@ final: prev:
     meta.homepage = "https://github.com/shaunsingh/moonlight.nvim/";
   };
 
-  render-markdown = buildVimPlugin {
-    pname = "render-markdown";
+  render-markdown-nvim = buildVimPlugin {
+    pname = "render-markdown-nvim";
```

Comment:
This one should stay `render-markdown.nvim`

Reply:
Fixed. :)